### ISSUE TITLE
Lint issues already posted to Github PR do not display in terminal

### ISSUE THREAD

**Issue by supremebeing7** [prontolabs/pronto#256]
We're using Pronto with a couple runners on Travis using the Github integration.

After one build, pronto posts comments on the Github PR with all of the lint issues it found **and** it shows the lint issues in the travis build log.

However, if I re-run the build:
- It **does not** post duplicate comments on the Github PR 👍 
- It also **does not** display the lint issues in the travis build log 👎 

Is this the expected behavior? And if so, is it configurable? 

It threw me off seeing pronto exiting with a nonzero status code, but not displaying any lint issues found.

**Comment by mmozuras** [comment]
@supremebeing7 it's not the expected behavior. What command do you use in your build? Did I understand correctly, you use something like `pronto run -r text github_pr`?

**Comment by supremebeing7** [comment]
`pronto run -f github_pr --exit-code`

Is it the `-f`/`--formatter` flag that's causing this? Will the `-r`/`--runner` flag post to the github PR *and* display in the build log?

**Comment by mmozuras** [comment]
@supremebeing7 I mistyped, it was supposed to be `-f`, not `-r` 🙃

And what's in your `.pronto.yml`?

**Comment by supremebeing7** [comment]
We don't have one, just using the defaults.

**Comment by mmozuras** [comment]
@supremebeing7 without `verbose` or `text` (formatter), I'm not sure what you were seing in the log. Could you post an example from your travis build log?

**Comment by supremebeing7** [comment]
@mmozuras Sure. This is one where pronto
```
4.04s$ lib/ci/test_runner.sh
if [ "$LINTING" = 1 ]
then bundle exec pronto run -f github_pr --exit-code
else bundle exec rake knapsack:rspec
fi
2 Pronto messages posted to GitHub
The command "lib/ci/test_runner.sh" exited with 3.
```
We have the test command in a separate shell script, so that's what you see on the first line. We also have 3 test runners on travis, and linting only happens on one (hence the conditional). You can also ignore the readline warnings, those have been present for ages and happen anytime the app boots in travis.

**Comment by supremebeing7** [comment]
Oh boy, sorry, I think I gave inaccurate info in my first post.
>After one build, pronto posts comments on the Github PR with all of the lint issues it found and it shows the lint issues in the travis build log.

I just checked the first build for a branch, and it ***does not*** show the lint issues in the travis build log at all. It ***only*** posts as comments on Github PR. It would be ideal if it would post to Github PR *and* show the lint errors in the build log.

With that in mind, is the solution just to run it with two different formatters, `github_pr` and whatever runs by default? Looking through the docs again, it makes it sound like my specific circumstance is actually expected behavior.
